src/refactor_nrepl/middleware.clj

```diff
@@ -157,8 +157,8 @@
   (delay
    (require-and-resolve 'refactor-nrepl.rename-file-or-dir/rename-file-or-dir)))
 
-(defn- rename-file-or-dir-reply [{:keys [transport old-path new-path] :as msg}]
-  (reply transport msg :touched (@rename-file-or-dir old-path new-path)
+(defn- rename-file-or-dir-reply [{:keys [transport old-path new-path ignore-errors] :as msg}]
+  (reply transport msg :touched (@rename-file-or-dir old-path new-path (= ignore-errors "true"))
```

Comment:
Not sure if I should compare against `true` or `"true"` here?

...Checking out the codebase, I see a handful of `"true"` handlings. But there's also a boolean coercion in config.clj which makes me hesitate

Reply:
This is correct, the stuff that's in the `msg` hasn't yet been coerced to boolean.

I suppose we could've done a bit better about consistently doing coercions in this ns (like you're doing now) instead of passing the `msg` on as is.